cvat-core/src/server-proxy.ts

```diff
@@ -855,24 +856,16 @@ function exportDataset(instanceType: 'projects' | 'jobs' | 'tasks') {
             ...(name ? { filename: name } : {}),
             format,
         };
-
         return new Promise<string | void>((resolve, reject) => {
             async function request() {
                 Axios.get(baseURL, {
                     params,
                 })
                     .then((response) => {
-                        const isCloudStorage = targetStorage.location === StorageLocation.CLOUD_STORAGE;
-                        const { status } = response;
-
-                        if (status === 202) {
-                            setTimeout(request, 3000);
-                        } else if (status === 201) {
-                            params.action = 'download';
-                            resolve(`${baseURL}?${new URLSearchParams(params).toString()}`);
-                        } else if (isCloudStorage && status === 200) {
-                            resolve();
+                        if (response.status === 202) {
+                            resolve(response.data.rq_id);
                         }
+                        resolve();
```

Comment:
Are there any cases, when success response does not include `rq_id`?

Reply:
Yes, in case of export and backups, if the export is finished but not downloaded, new export is not started and we get 200 response without `rq_id` (the same thing for other same comments)

Reply:
So, then something is not working here.
For example:

- serverProxy.backupTask or backupProject returns string or undefined.
- it means Task.backup, and Project.backup also returns string or undefined
- string or undefined is used to start listening (in case with undefined, it seems there is typing error)

![image](https://github.com/cvat-ai/cvat/assets/40690378/943cfa6c-a1d0-427b-a61d-496f886ee93e)

Reply:
Perhaps the same for export, but I did not check